packages/fx-core/src/plugins/resource/bot/constants.ts

```diff
@@ -276,3 +276,8 @@ export class BotBicep {
   static readonly hostName: string = "provisionOutputs.botOutput.value.validDomain";
   static readonly webAppEndpoint: string = "provisionOutputs.botOutputs.value.botWebAppEndpoint";
 }
+
+export class ArmTemplateVersion {
+  static readonly BicepVersion = "0.4.1008";
+  static readonly SchemaVersion = "0.1.0";
```

Comment:
Same for SchemaVersion, could we put this in a central place, so that we would not change it in several files next time.